platform/ext/target/nordic_nrf/common/nrf54l/config.cmake

```diff
@@ -8,7 +8,7 @@
 
 include(${PLATFORM_PATH}/common/core/config.cmake)
 
-set(SECURE_UART30                       ON         CACHE BOOL      "Enable secure UART" FORCE)
+set(SECURE_UART20                       ON         CACHE BOOL      "Enable secure UART" FORCE)
```

Comment:
Just for my understanding. Changing UART30=>UART20 here is just a default, or is this changed intended for all device types in nRF54L series?

Reply:
Yeah this just changes the default, while we still set the secure uart from dts downstream.

Reply:
Actually not correct. Had to change this upstream to get 54l15 to work